lambdas/retrieve-cri-oauth-access-token/src/main/java/uk/gov/di/ipv/core/retrievecrioauthaccesstoken/RetrieveCriOauthAccessTokenHandler.java

```diff
@@ -131,17 +132,18 @@ public APIGatewayProxyResponseEvent handleRequest(
             BearerAccessToken accessToken =
                     credentialIssuerService.exchangeCodeForToken(
                             request, credentialIssuerConfig, apiKey);
-            SignedJWT verifiableCredential =
+            List<SignedJWT> verifiableCredentials =
                     credentialIssuerService.getVerifiableCredential(
                             accessToken, credentialIssuerConfig, apiKey);
 
-            verifiableCredentialJwtValidator.validate(
-                    verifiableCredential, credentialIssuerConfig, userId);
+            for (SignedJWT vc : verifiableCredentials) {
```

Comment:
Just curious why do we get multiple VCs in one go from the app?

Reply:
I'm not sure of the specific reason why, and I have a feeling that for now at least it will always be 1 item. But its listed in their documentation that this is returned as an array - https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/3141435393/DCMAW+-+Verifiable+Credential+Response+-+Drivers+License